http/src/main/resources/init-resources/shutdown.sh

```diff
@@ -22,10 +22,6 @@ export RSTUDIO_DOCKER_IMAGE=$(rstudioDockerImage)
 export RSTUDIO_SERVER_NAME=$(rstudioServerName)
 export SHOULD_DELETE_JUPYTER_DIR=$(shouldDeleteJupyterDir)
 
-# If RStudio is installed, cleanly shut it down
```

Comment:
We do not do this for jupyter, and I did not run into issues when testing on my BEE